js/main.js

```diff
@@ -1,31 +1,51 @@
 var GOOGLE_GEOLOCATION_API_KEY = 'AIzaSyDOtBrAgfz68KEzcoArjq5MK9mNh6Uq1V8'
 
-const ETH_NODE_URL = 'https://ropsten.infura.io/wUiZtmeZ1KwjFrcC8zRO';
+var ETH_NODE_URL = 'https://ropsten.infura.io/wUiZtmeZ1KwjFrcC8zRO';
 //const ETH_NODE_URL = 'https://mainnet.infura.io/wUiZtmeZ1KwjFrcC8zRO';
 
-let web3Provider = new Web3
+var web3Provider = new Web3
     .providers
     .HttpProvider(ETH_NODE_URL);
-let web3 = new Web3(web3Provider);
+var web3 = new Web3(web3Provider);
 window.contractInstance = new web3.eth.Contract(window.contractArtifact.abi, window.contractArtifact.address);
-let weiRaised = null;
+var weiRaised = null;
 
 function numberWithCommas(number) {
   var parts = number.toString().split(".");
   parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
   return parts.join(".");
 }
 
+var $ethRaised = $("#eth-raised");
 function updateEthRaised() {
   window.contractInstance.methods.weiRaised().call(function(error, results) {
     if(!error) {
       weiRaised = results;
-      let ethRaised = web3.utils.fromWei(weiRaised, 'ether');
-      $(".eth-raised")[0].innerHTML = numberWithCommas(ethRaised);
+      var ethRaisedValue = web3.utils.fromWei(weiRaised, 'ether');
+      increaseWithAnimation($ethRaised, ethRaisedValue)
     }
   });
 }
 
+var ANIMATION_DURATION = 2000;
+var PULSE_DURATION = 40;
+function increaseWithAnimation($element, value) {
+  var currentValue = Number($element.text().replace(/,/g , ''));
+  var newValue = currentValue + Math.floor(value);
```

Comment:
target value should be ethRaisedValue (not the sum of ethRaisedValue and current)